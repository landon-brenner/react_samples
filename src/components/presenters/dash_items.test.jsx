// Snapshot test for DashItems component. Includes MUI shallow rendering
import React from 'react';
import DashItems from './dash_items';
import { createShallow } from '@material-ui/core/test-utils';

describe('<DashItems />', () => {
  it('matches snapshot', () => {
    const wrapper = createShallow(<DashItems />)
    expect(wrapper).toMatchSnapshot();
  });
});
