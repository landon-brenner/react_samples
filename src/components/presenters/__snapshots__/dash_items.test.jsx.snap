// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashItems /> matches snapshot 1`] = `[Function]`;
